app/src/test/java/com/example/diceroller/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package com.example.diceroller;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
```

Comment:
is this test needed? is it part of the codelab?

Reply:
Correct. This test is a part of a codelab.